docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -28,7 +28,7 @@ Running the tests requires a Django settings module that defines the
 databases to use. To make it easy to get started, Django provides and uses a
 sample settings module that uses the SQLite database. To run the tests:
 
-.. code-block:: bash
+.. code-block:: console
```

Comment:
I'd add `.. highlight:: console` to top of the file then switch to use this style:

```rst
[...] To run the tests::  # notice two :

    $ git clone https://github.com/django/django.git django-repo
    cd django-repo/tests
```